WpfAnimatedGif/ImageBehavior.cs

```diff
@@ -300,11 +302,31 @@ private static void AnimatedSourceChanged(DependencyObject o, DependencyProperty
             {
                 imageControl.Loaded += ImageControlLoaded;
                 imageControl.Unloaded += ImageControlUnloaded;
+                imageControl.IsVisibleChanged += VisibilityChanged;
+
                 if (imageControl.IsLoaded)
                     InitAnimationOrImage(imageControl);
             }
         }
 
+        private static void VisibilityChanged(object sender, DependencyPropertyChangedEventArgs e)
+        {
+            if (sender is Image img && img.IsLoaded)
+            {
+                var controller = GetAnimationController(img);
+                if (controller != null)
+                {
+                    if ((bool)e.NewValue)
+                    {
+                        controller.GotoFrame(0);
+                        controller.Play();
+                    }
+                    else
+                        controller.Pause();

```

Comment:
Minor issue: while I'm not against omitting braces in general, I prefer to keep the `if` and `else` blocks consistent. Could you please add braces there?